docs/install/Knative-with-Contour.md

```diff
@@ -0,0 +1,174 @@
+---
+title: "Knative Install using Contour on a Kubernetes Cluster"
+linkTitle: "Contour"
+weight: 10
+type: "docs"
+---
+
+Learn how to deploy Contour and Knative to your Kubernetes cluster.
+
+[Contour](https://projectcontour.io/) is an open source Kubernetes ingress controller providing the control plane for the Envoy edge and service proxy.
+
+## Before you begin
+
+Knative requires a Kubernetes cluster v1.15 or newer, as well as a compatible
+`kubectl`.  This guide assumes that you've already created a Kubernetes cluster,
+and that you are using bash in a Mac or Linux environment; some commands will
+need to be adjusted for use in a Windows environment.
+
+## Installing Knative Serving
+
+First, let's install Knative to manage our serverless applications.
+
+The following commands install the Knative Serving components:
+
+1.  To install Knative, first install the CRDs by running the following `kubectl apply`
+    command. This prevents race conditions during the install, which cause intermittent errors:
+
+        kubectl apply --filename https://github.com/knative/serving/releases/download/{{< version >}}/serving-crds.yaml
+
+2.  To complete the install of Knative and its dependencies, next run the
+    following `kubectl apply` command:
+
+        kubectl apply --filename https://github.com/knative/serving/releases/download/{{< version >}}/serving-core.yaml
+
+3.  Monitor the `knative-serving` namespaces and wait until all of the pods come up with a
+    `STATUS` of `Running`:
+
+    ```
+    kubectl get pods -w --all-namespaces
+    ```
+
+### Installing Contour for Knative
+
+This next step will install both Contour (in the namespace `projectcontour`) and an additional
```

Comment:
Replace line 44 and 45 with these lines:

Contour is installed in the namespace `projectcontour` with an additional controller in `knative-serving` to bridge the two.

To install Contour, enter the following command: